### PR TITLE
map some biome things and misc important things

### PR DESCRIPTION
**Pull request by eristhea**
a perusal of worldgen and a few things I noticed while porting.

### CODE REVIEW

File: mappings/net/minecraft/world/biome/OverworldBiomeCreator.mapping [hunk 1]
```diff
@@ -63,9 +71,10 @@ CLASS net/minecraft/unmapped/C_zulhefzk net/minecraft/world/biome/OverworldBiome
 	METHOD m_lklnlnru createDarkForest (Lnet/minecraft/unmapped/C_pzdchrcy;Lnet/minecraft/unmapped/C_pzdchrcy;)Lnet/minecraft/unmapped/C_orlkpefs;
 		ARG 0 features
 		ARG 1 carvers
-	METHOD m_lsekqdan createCherryGrove (Lnet/minecraft/unmapped/C_pzdchrcy;Lnet/minecraft/unmapped/C_pzdchrcy;Z)Lnet/minecraft/unmapped/C_orlkpefs;
+	METHOD m_lsekqdan createMeadowOrCherryGrove (Lnet/minecraft/unmapped/C_pzdchrcy;Lnet/minecraft/unmapped/C_pzdchrcy;Z)Lnet/minecraft/unmapped/C_orlkpefs;
```
Comment (EnnuiL) [inline mappings/net/minecraft/world/biome/OverworldBiomeCreator.mapping:74]: oh dear, did i miss a spot?

**Review by EnnuiL — APPROVED** [review]
Excellent!

File: mappings/net/minecraft/world/biome/OverworldBiomeCreator.mapping (hunk 1 above)
Comment (eristhea) [reply mappings/net/minecraft/world/biome/OverworldBiomeCreator.mapping:74]: not sure what happened here!

File: mappings/net/minecraft/recipe/DecoratedPotRecipe.mapping [hunk 2]
```diff
@@ -1 +1,4 @@
 CLASS net/minecraft/unmapped/C_wcabtfez net/minecraft/recipe/DecoratedPotRecipe
+	METHOD m_pchpzfnb createStack (Lnet/minecraft/unmapped/C_snnddtnv$C_gxhwrzck;)Lnet/minecraft/unmapped/C_sddaxwyk;
+		COMMENT Creates a decorated pot stack with the provided sherds as NBT.
```
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/recipe/DecoratedPotRecipe.mapping:3]:
```suggestion
		COMMENT Creates a decorated pot stack with the provided shards as NBT.
```

File: mappings/net/minecraft/world/ChunkSerializer.mapping [hunk 3]
```diff
@@ -11,23 +11,39 @@ CLASS net/minecraft/unmapped/C_bsqbntcq net/minecraft/world/ChunkSerializer
...
 		ARG 0 lists
 	METHOD m_gcmtnulo getChunkType (Lnet/minecraft/unmapped/C_hhlwcnih;)Lnet/minecraft/unmapped/C_kogtzhzt$C_lcffkbeg;
 		ARG 0 nbt
+	METHOD m_iuzckbsr (Ljava/lang/String;)Ljava/util/Optional;
+		ARG 0 blockId
 	METHOD m_mxkjjsvz getList (Lnet/minecraft/unmapped/C_hhlwcnih;Ljava/lang/String;)Lnet/minecraft/unmapped/C_hqhtfwlr;
 		ARG 0 nbt
 		ARG 1 key
+	METHOD m_nxqmhxhy (Lnet/minecraft/unmapped/C_ynrszrtu;Lnet/minecraft/unmapped/C_ncpywfca;J)Z
+		ARG 2 l
```
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/world/ChunkSerializer.mapping]:
```suggestion
		ARG 2 pos
```
why is this one just `l`?

**Review by supersaiyansubtlety — CHANGES_REQUESTED** [review]

File: mappings/net/minecraft/recipe/DecoratedPotRecipe.mapping (hunk 2 above)
Comment (EnnuiL) [reply mappings/net/minecraft/recipe/DecoratedPotRecipe.mapping:3]: No, it's sherds

Comment (EnnuiL) [reply mappings/net/minecraft/recipe/DecoratedPotRecipe.mapping:3]: Context: In an archeological context, [sherds](https://en.wiktionary.org/wiki/sherd#English) is used over shards; This is what Mojang is using to name the items

File: mappings/net/minecraft/world/ChunkSerializer.mapping (hunk 3 above)
Comment (eristhea) [reply mappings/net/minecraft/world/ChunkSerializer.mapping]: you caught me being lazy :p
that's a lambda and I didn't verify if it was actually a position or not
will check in a bit!

Comment (eristhea) [reply mappings/net/minecraft/world/ChunkSerializer.mapping]: it is indeed a position, thanks!

File: mappings/net/minecraft/recipe/DecoratedPotRecipe.mapping (hunk 2 above)
Comment (supersaiyansubtlety) [reply mappings/net/minecraft/recipe/DecoratedPotRecipe.mapping:3]: Oh, TIL

**Review by supersaiyansubtlety — APPROVED** [review]

File: mappings/net/minecraft/world/biome/OverworldBiomeCreator.mapping [hunk 4]
```diff
@@ -11,9 +11,17 @@ CLASS net/minecraft/unmapped/C_zulhefzk net/minecraft/world/biome/OverworldBiome
...
+		ARG 7 music
 	METHOD m_czonisgu createMangroveSwamp (Lnet/minecraft/unmapped/C_pzdchrcy;Lnet/minecraft/unmapped/C_pzdchrcy;)Lnet/minecraft/unmapped/C_orlkpefs;
 		ARG 0 features
 		ARG 1 carvers
+	METHOD m_eepimsva create (ZFFLnet/minecraft/unmapped/C_ypgfhxmz$C_ooyakwqo;Lnet/minecraft/unmapped/C_qbjpvmph$C_bwxzcdhs;Lnet/minecraft/unmapped/C_usodapnd;)Lnet/minecraft/unmapped/C_orlkpefs;
+		ARG 0 hasPrecipitation
+		ARG 1 temperate
+		ARG 2 downfall
+		ARG 3 spawnSettings
+		ARG 4 generationSettings
```
Comment (EnnuiL) [inline mappings/net/minecraft/world/biome/OverworldBiomeCreator.mapping:23]: oh, shouldn't these be specifically be suffixed with "builder"?

**Review by EnnuiL — CHANGES_REQUESTED** [review]

File: mappings/net/minecraft/world/biome/OverworldBiomeCreator.mapping (hunk 4 above)
Comment (eristhea) [reply mappings/net/minecraft/world/biome/OverworldBiomeCreator.mapping:23]: I personally wouldn't, but if someone else says yes I'll go for it

**Pull request merged**